jsonrpc/src/methods/beacon.json

```diff
@@ -184,7 +187,10 @@
     ],
     "result": {
       "$ref": "#/components/contentDescriptors/LocalContentResult"
-    }
+    },
+    "errors":[{
```

Comment:
We should have the same error added for `portal_beaconRecursiveFindContent`

Reply:
Have added